scim_rails.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = "~> 2.4"
   s.add_dependency "rails", ">= 5.0", "< 6.1"
+  s.add_runtime_dependency "jwt", "~> 2.2"
```

Comment:
I'm testing this in our core application and am running into a conflict with using the latest version of 2.2. What do you think about using "~> 1.5.1"? I'm going to push it up real quick and test if everything plays nice after that

Reply:
Yep, that made it install seamless into our current environment